packages/pros/src/devices/smart/motor.rs

```diff
@@ -217,9 +190,13 @@ impl Motor {
         Ok(())
     }
 
-    //TODO: Test this, as im not entirely sure of the actual implementation
+    pub fn brake_mode(&self) -> Result<BrakeMode, MotorError> {
+        unsafe { pros_sys::motor_get_brake_mode(self.port.index()).try_into() }
+    }
+
+    // TODO: Test this, as im not entirely sure of the actual implementation
     /// Get the current state of the motor.
-    pub fn get_state(&self) -> Result<MotorState, MotorError> {
+    pub fn state(&self) -> Result<MotorState, MotorError> {
```

Comment:
We aren't exactly at the point of caring too much about semver bumps, since we're pre-1.0 and make some pretty heavy breaking changes often. Not sure if we have enough people currently using to care about deprecating for now, since there are certainly more breaking changes to come.

Reply:
I don't think that it's necessary to deprecate it, but I also wouldn't be against it. Even if it was deprecated this pr would still need to bump a semver major version because of other breaking changes.